src/coreComponents/python/CMakeLists.txt

```diff
@@ -30,8 +30,13 @@ if ( Python3_EXECUTABLE )
                          WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
                        )
 
-     add_custom_target( geosx_python_tools
-                        DEPENDS ${GEOSX_PYTHON_TOOLS_BINS} )
+     if( ENABLE_PYGEOSX )
+          add_custom_target( geosx_python_tools
+                             DEPENDS pygeosx ${GEOSX_PYTHON_TOOLS_BINS} )
+     else()
+          add_custom_target( geosx_python_tools
+                             DEPENDS ${GEOSX_PYTHON_TOOLS_BINS} )
+     endif()
```

Comment:
@TotoGaz @rrsettgast - I've added pygeosx as a dependency of geosx_python_tools when PYGEOSX is active.  I've also removed the (now uneccesary) error messages to the user in the setup script instructing to manually run this step if they missed it.